CONTRIBUTING.md

```diff
@@ -7,4 +7,8 @@ Contributions to fabricate are welcome! However, we do have a couple dependencie
 3. Run `mongod --dbpath path/to/db`
 4. Everything should be setup and you should be ready to contribute.
 
-A docker container for all of this is coming soon to make this process easier.
\ No newline at end of file
+Alternatively, if you have Docker and docker-compose installed, you may run:
```

Comment:
Maybe put it as the recommended method, so put it at the top. It should provide the most consistent results

Reply:
resolved in `11a5088`